code/datums/tutorial/xenomorph/xenomorph_basic.dm

```diff
@@ -0,0 +1,229 @@
+	if(human_dummy.health < (human_dummy.maxHealth / 2))
+		message_to_player("Don't harm the human!")
+		human_dummy.rejuvenate()
+
+/datum/tutorial/xenomorph/basic/proc/proceed_to_cap_phase()
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+
+	UnregisterSignal(human_dummy, COMSIG_MOB_TACKLED_DOWN)
+
+	ADD_TRAIT(human_dummy, TRAIT_KNOCKEDOUT, TRAIT_SOURCE_TUTORIAL)
+	ADD_TRAIT(human_dummy, TRAIT_FLOORED, TRAIT_SOURCE_TUTORIAL)
+	xeno.melee_damage_lower = 0
+	xeno.melee_damage_upper = 0
+	message_to_player("Well done. Under normal circumstances, you would have to keep tackling the human to keep them down, but for the purposes of this tutorial they will stay down forever.")
+	addtimer(CALLBACK(src, PROC_REF(cap_phase)), 10 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/cap_phase()
+	var/obj/effect/alien/resin/special/eggmorph/morpher = new(loc_from_corner(2,2), GLOB.hive_datum[XENO_HIVE_TUTORIAL])
+	morpher.stored_huggers = 1
+	add_to_tracking_atoms(morpher)
+	add_highlight(morpher, COLOR_YELLOW)
+	message_to_player("In the south west is an egg morpher. Click the egg morpher to take a <b>facehugger</b>.")
+	RegisterSignal(xeno, COMSIG_XENO_TAKE_HUGGER_FROM_MORPHER, PROC_REF(take_facehugger_phase))
+
+/datum/tutorial/xenomorph/basic/proc/take_facehugger_phase(source, hugger)
+	SIGNAL_HANDLER
+	UnregisterSignal(xeno, COMSIG_XENO_TAKE_HUGGER_FROM_MORPHER)
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/effect/alien/resin/special/eggmorph, morpher)
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+	add_to_tracking_atoms(hugger)
+	remove_highlight(morpher)
+
+	add_highlight(hugger, COLOR_YELLOW)
+	message_to_player("This is a facehugger, highlighted in yellow. Pick up the facehugger by clicking it.")
+	message_to_player("Stand next to the downed human and click them to apply the facehugger. Or drop the facehugger near them to see it leap onto their face automatically.")
+	RegisterSignal(human_dummy, COMSIG_HUMAN_IMPREGNATE, PROC_REF(nest_cap_phase))
+
+/datum/tutorial/xenomorph/basic/proc/nest_cap_phase()
+	SIGNAL_HANDLER
+	TUTORIAL_ATOM_FROM_TRACKING(/mob/living/carbon/human, human_dummy)
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/item/clothing/mask/facehugger, hugger)
+	UnregisterSignal(human_dummy, COMSIG_MOB_TAKE_DAMAGE)
+	UnregisterSignal(human_dummy, COMSIG_HUMAN_IMPREGNATE)
+	remove_highlight(hugger)
+
+	message_to_player("We should nest the infected human to make sure they don't get away.")
+	message_to_player("Humans cannot escape nests without help, and the nest will keep them alive long enough for our new sister to burst forth.")
+	addtimer(CALLBACK(src, PROC_REF(nest_cap_phase_two)), 10 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/nest_cap_phase_two()
+
+	new /turf/closed/wall/resin/tutorial(loc_from_corner(8,0))
+	new /turf/closed/wall/resin/tutorial(loc_from_corner(8,1))
+	new /turf/closed/wall/resin/tutorial(loc_from_corner(9,1))
```

Comment:
should these be changeturfed to avoid weirdness? i suppose it gets changeturfed back after the tutorial reservation is released so it doesn't much matter